ftl/common/cache.py

```diff
@@ -107,6 +111,8 @@ def _tag(self, cache_key, repo=None):
             tag=hashlib.sha256(fingerprint).hexdigest()))
 
     def Get(self, cache_key):
+        if not self._should_cache:
```

Comment:
Can you add a logging statement in both of these if statements?

Reply:
done